libcfnet/client_code.c

```diff
@@ -748,7 +748,8 @@ bool CopyRegularFileNet(const char *source, const char *dest, off_t size,
 {
     assert(conn != NULL);
 
-    char *buf, workbuf[CF_BUFSIZE], cfchangedstr[265];
+    char buf[CF_BUFSIZE + sizeof(int)]; /* Note CF_BUFSIZE not buf_size !! */
```

Comment:
The `sizeof(int)` makes me a little worried, as it can be either 4 or 8 Bytes

Reply:
Yeah... Do you know what it is there for? Please create a ticket for this if you cannot spend the extra time on it now.